src/StockAvailabilityChecker.php

```diff
@@ -56,15 +56,11 @@ public function check(PurchasableEntityInterface $entity, $quantity, Context $co
     // Get locations.
     $locations = $stock_config->getLocationList($entity);
 
```

Comment:
not sure why this was changed? same logic as before, but maybe reads better. If was a good reason for it could have also moved the "// Get locations."  part to be under the always in stock check, as it is not used before that point. 
Not a problem but if not necessary less code to review and less chances of merge conflicts.